benchmark/benchmark.py

```diff
@@ -64,6 +64,12 @@ def _compare_client_results(self, client_run, self_analyzer, baseline_analyzer):
                 logger.info('CPU Cycles Per Operation metric is not configured for this benchmark')
                 continue
             self_analyzer_res[name] = self_getter()
+            if self_analyzer_res[name] is None:
+                with open("/proc/sys/kernel/perf_event_paranoid") as f:
+                    priv_level = int(f.read())
+                    if (priv_level >= 1):
+                        logger.warning('Perf must be ran with correct privileges to extract CPU related metrics, current privilege level:', priv_level)
```

Comment:
this line throws like
```
TypeError: not all arguments converted during string formatting
```

```suggestion
                        msg = '''Perf must be run with correct privileges to extract CPU
related metrics. or you could set %s to 1, which is %d now'''
                        logger.warning('%s. %s is %d', msg, paranoid_path, paranoid_level)
```

where i assume a `paranoid_path` variable is defined to avoid repeating.